canvas_modules/common-canvas/src/common-properties/controls/expression/expression.jsx

```diff
@@ -88,10 +88,12 @@ class ExpressionControl extends React.Component {
 		 * The ARIA attributes "aria-label" are not valid for the element <span> with implicit ARIA
 		 * role "generic"
 		*/
-		const placeholderElement = document.querySelector(".properties-expression-editor-wrapper .cm-placeholder");
-		if (placeholderElement) {
-			placeholderElement.removeAttribute("aria-label");
-		}
+		const placeholderElements = document.querySelectorAll(".properties-expression-editor-wrapper .cm-placeholder");
+		placeholderElements.forEach((placeholderElement) => {
```

Comment:
I don't think this is a good solution.  This would do the loop on every expression in the browser.  Shouldn't it only remove it for the specific react element?

Reply:
Yes, I will look for a better solution meanwhile waiting for response from codemirror team over [here](https://discuss.codemirror.net/t/accessibility-violation-in-placeholder/9165/3). If they remove `aria-label` from `span`  then I will remove this piece of code.

Reply:
@srikant-ch5 `@codemirror/view` latest version is available - https://www.npmjs.com/package/@codemirror/view

Reply:
@nmgokhale updated PR to use latest version of `codemirror/view`.